### PR TITLE
Pylint results

### PR DESCRIPTION
**Pull request by camenpihor**

### CODE REVIEW

**Comment by camenpihor** [comment]
Does anyone have suggestions on how to improve coverage here (since I didn't add significant code that I can test)

File: bambi/results.py [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+
+import arviz as az
 import numpy as np
-import matplotlib.pyplot as plt
+import pandas as pd
+from bambi.external.six import string_types
+
 from .utils import listify
 
+try:
```
Comment (aloctavodia) [inline bambi/results.py]: I removed all this code in  #169

Comment (aloctavodia) [inline bambi/results.py]: I removed this code in

Comment (ColCarroll) [reply bambi/results.py]: (specifically, lines 11-19 can be deleted)

**Comment by ColCarroll** [comment]
Don't worry about the coverage -- it can still merge, we just have the annoying :x:

**Comment by ColCarroll** [comment]
:penguin:

**Pull request merged**